x-pack/plugins/apm/public/components/shared/service_icons/icon_popover.tsx

```diff
@@ -46,13 +51,17 @@ export function IconPopover({
       anchorPosition="downCenter"
       ownFocus={false}
       button={
-        <EuiButtonEmpty onClick={onClick} data-test-subj={`popover_${title}`}>
+        <StyledPanel
```

Comment:
The result looks a bit different than expected on my screen (Chrome) compared to the Storybook example 

<img width="477" alt="CleanShot 2022-01-11 at 12 52 55@2x" src="https://user-images.githubusercontent.com/4104278/148937992-9157e3e0-2ec8-472f-ba7e-071ce6d4e42f.png">

<img width="209" alt="CleanShot 2022-01-11 at 12 53 57@2x" src="https://user-images.githubusercontent.com/4104278/148938126-5cd00089-d8a8-411d-a6d8-d9be8b1565ee.png">